src/vue/backend/src/App.vue

```diff
@@ -1,12 +1,61 @@
 <template>
-  <idx-block className="app">
```

Comment:
Test code

Reply:
This file should not be modified at all, since it is test code. Otherwise it will cause merge conflicts when the non-test code is merged

Reply:
I put everything back in code, but it now has a 4-space indent rather than 2 :(